### PR TITLE
Update drupal.js

### PR DESCRIPTION
**Pull request by heddn**
as of Drupal 9.2, webp support is natively supported in Drupal core. Clean-up the docs to make this more clear and not recommend a contrib module. Version of Drupal 9 prior to 9.2 aren't supported any longer, so streamlined docs seems like the right direction at this point. There is still a bit of an edge case (outlined in https://www.drupal.org/project/drupal/issues/3213491#comment-14473712) why someone might want to still use the contrib module, but that is very time boxed and nuanced reason. Using plain webp is probably enough for most sites.

### CODE REVIEW

File: packs/drupal.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const UIStrings = {
...
   /** Additional description of a Lighthouse audit that tells the user how they can improve image loading by using webp in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  'modern-image-formats': 'Consider installing and configuring [a module to leverage WebP image formats](https://www.drupal.org/project/project_module?f%5B0%5D=&f%5B1%5D=&f%5B2%5D=&f%5B3%5D=&f%5B4%5D=sm_field_project_type%3Afull&f%5B5%5D=&f%5B6%5D=&text=webp&solrsort=iss_project_release_usage+desc&op=Search) in your site. Such modules automatically generate a WebP version of your uploaded images to optimize loading times.',
+  'modern-image-formats': 'Consider installing and configuring [WebP image formats with a Convert image style](https://www.drupal.org/docs/core-modules-and-themes/core-modules/image-module/working-with-images#styles) on your site.',
```
Comment (connorjclark) [inline packs/drupal.js:10]: can we drop this bit now?
```suggestion
  'modern-image-formats': 'Consider configuring [WebP image formats with a Convert image style](https://www.drupal.org/docs/core-modules-and-themes/core-modules/image-module/working-with-images#styles) on your site.',
```

Comment (connorjclark) [reply packs/drupal.js:10]: also we may want to wrap "Convert" in backticks, this prevents the word from being translated. Any maybe lowercase it (or does it show as uppercase in the actual UI)? Also assuming that UI isn't itself translated...if it is, ignore all this

Comment (heddn) [reply packs/drupal.js]: It is translated. And it is uppercase in the UI: https://git.drupalcode.org/project/drupal/-/blob/9.4.x/core/modules/image/src/Plugin/ImageEffect/ConvertImageEffect.php#L14

**Review by connorjclark — APPROVED** [review]
Thanks!

**Pull request merged**